src/test/java/teammates/test/cases/ui/StudentHomePageActionTest.java

```diff
@@ -84,8 +85,9 @@ public void testExecuteAndPostProcess() throws Exception {
                                           + studentWithoutCourses.googleId,
                                           r.getDestinationWithParams());
         assertFalse(r.isError);
-        AssertHelper.assertContainsRegex("Welcome stranger :-){*}use the new Gmail address.",
-                                          r.getStatusMessage());
+        AssertHelper.assertContainsRegex(
+                "Ooops! Your Google account is not known to TEAMMATES" + "{*}use the new Gmail address.",
```

Comment:
Here it doesn't need to be separated, just `...TEAMMATES{*}use...`. Similarly for few lines above.